boards/arm/mimxrt1160_evk/mimxrt1160_evk_cm4.dts

```diff
@@ -36,3 +36,21 @@
 &lpi2c1 {
 	status = "okay";
 };
+
+&lpi2c5 {
+	status = "okay";
+
+	fxos8700@1f {
+		compatible = "nxp,fxos8700";
+		reg = <0x1f>;
+		label = "FXOS8700";
+
+		/* Two zero ohm resistors (R256 and R270) isolate sensor
+		 * interrupt gpios from the soc and are unpopulated by default.
+		 * Note that if you populate them, they conflict with camera and
+		 * ethernet PHY reset signals.
+		 */
+		int1-gpios = <&gpio11 14 GPIO_ACTIVE_LOW>;
+		int2-gpios = <&gpio11 15 GPIO_ACTIVE_LOW>;
```

Comment:
Move this to the shared `boards/arm/mimxrt1160_evk/mimxrt1160_evk.dtsi` so you don't have to duplicate for each core. Ditto for rt1170